pkg/ccl/changefeedccl/sink_webhook.go

```diff
@@ -241,6 +260,26 @@ func (s *webhookSink) Dial() error {
 	return nil
 }
 
+func (s *webhookSink) sendMessageWithRetries(ctx context.Context, reqBody []byte) error {
+	var backoffTime time.Duration
+	var i uint
+	var err error
+	for i = 0; i <= s.retryCfg.maxRetries; i++ {
+		time.Sleep(backoffTime)
```

Comment:
sleep calls like this make it hard to test.  Just fyi.

Reply:
changed to use `retry.WithMaxAttemps`